apps/tasktracker/providers/pivotal.go

```diff
@@ -183,12 +184,20 @@ func (c *PivotalConnection) GetTaskUrl(ticketKey string) string {
 	return fmt.Sprintf("https://www.pivotaltracker.com/story/show/%v", ticketKey)
 }
 
+// CleanTickets ...
+func (c *PivotalConnection) CleanTickets(ticketKeys []string) {
```

Comment:
If it is an internal method then we should make the first letter small, i.e., `cleanTickets`.

Reply:
done